ntp-proto/src/peer.rs

```diff
@@ -94,11 +94,21 @@ impl Reach {
 
 #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
 pub struct SystemSnapshot {
-    /// May be updated by local_clock
+    /// Desired poll interval
     pub poll_interval: PollInterval,
-    /// A constant at runtime
+    /// Precision of the local clock
     pub precision: NtpDuration,
-    /// May be updated by clock_update
+    /// Log of the precision of the local clock
+    pub log_precision: i8,
```

Comment:
maybe rename to `clock_log_position` or `clock_position_log`?

Reply:
also, how should this value be interpreted? why are negative numbers used in the tests?

Reply:
On second thought, I think it is a premature optimization to add this right now. For future reference, it is the 2log (rounded to integer) of the previous argument, though I made a small mistake in filling it in in the tests.